source/infrastructure/lib/core-scheduler.ts

```diff
@@ -0,0 +1,159 @@
+import {LambdaToDynamoDB, LambdaToDynamoDBProps} from "@aws-solutions-constructs/aws-lambda-dynamodb";
+import {Aws, RemovalPolicy} from "aws-cdk-lib";
+import * as lambda from "aws-cdk-lib/aws-lambda";
+import * as cdk from "aws-cdk-lib";
+import * as dynamodb from "aws-cdk-lib/aws-dynamodb";
+import {Construct} from "constructs";
+import {Effect, PolicyStatement} from "aws-cdk-lib/aws-iam";
+import {NagSuppressions} from "cdk-nag";
+export interface InstanceSchedulerLambdaProps {
+
+  solutionVersion: string
```

Comment:
readonly - this was commented in the previous PR - please address all of those comments before re-requesting review: https://github.com/aws-solutions/aws-instance-scheduler/pull/374